clang/include/clang/Basic/DiagnosticFrontendKinds.td

```diff
@@ -134,6 +134,8 @@ def err_fe_no_pch_in_dir : Error<
     "no suitable precompiled header file found in directory '%0'">;
 def err_fe_action_not_available : Error<
     "action %0 not compiled in">;
+def err_fe_invalid_multiple_actions : Error<
+    "action %0 is specified, another action is not allowed: %1">;
```

Comment:
```suggestion
    "'%1' action ignored; '%0' action specified previously">;
```

Maybe something like this?  I dont think the original fits very well?  Also the single ticks around the actions is, IMO, better for readability.

Reply:
Thanks for the suggestion. I adopted it by swapping %1 and %0 ...